src/makeCalculator.js

```diff
@@ -4,7 +4,32 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate: function (func, num) {
+      func.call(this, num);
```

Comment:
Ensure that the `func` parameter passed to the `operate` function is always a valid function to avoid runtime errors.